docs/decisions/0001-namespacing-branches.rst

```diff
@@ -0,0 +1,84 @@
+Namespacing Branches and Tags
+#############################
+
+Status
+******
+
+Draft
+
+Context
+*******
+
+This ADR was written in response to this `Discourse thread`_.
+
+Currently, 2U continuous delevery tooling creates branches and tags
+in the repository "global namespace."  By that I mean that branches
+and tags have the form ``release`` or ``release-candidate-3729``.
+This is confusing because those branches and tags appear to be global,
+officially sanctioned by the project, and authoritative.
+
+A result of this is that the official named releases of the platform
+require a namespace specifier, for example,
+``open-release/olive.master``.
+
+This is confusing to most members of the community and reflects the
+prior stewardship arrangements for the project.
+
+It is not typical of open-source projects to allow participating
+organizations to push organization specific branches and tags to the
+canonical upstream.  However, we recommend not prohibiting this at this
+time.  There are two reasons for that:
+
+1. This would require more substantial changes to participant deployment
+   systems that would delay getting value from this initial change.
+2. The 2U branches and tags represent code that is battle-tested in
+   production at scale and are, thus, valuable sign posts for
+   community members.
+
+.. _Discourse thread: https://discuss.openedx.org/t/should-we-rename-the-release-branches/8827/7
+
+
+Decisions
+*********
+
+1. We will not forbid project participants with write access to
```

Comment:
Additionally, I don't know how many repos actively have these types of branches or tags, and whether or not it would make sense to limit this exception to a particular set of repos. This is just a thought, without much understanding of the implications of going either way, so feel free to ignore this if it isn't helpful.

Reply:
That idea also interests me, but I'd rather leave it to a future decision.

Reply:
@kdmccormick: I’m marking this resolved. Thanks.